packages/itwin/property-grid/src/components/PropertyGrid.tsx

```diff
@@ -303,6 +294,24 @@ export const PropertyGrid = ({
           }
         }
 
+        // Do any overrides on default menu options
+        if(defaultContextMenuOptions?.size && defaultContextMenuOptions.size > 0 ) {
+          for(const key of Object.values(PropertyGridDefaultContextMenuKey)) {          
+            const overrides = defaultContextMenuOptions?.get(key);
+            const itemIndex = items.map((item) => item.key).indexOf(key);
+            items[itemIndex] = { ...items[itemIndex], ...overrides};
+          }
+        }
+
+        // Verify all existing options are valid, and if not remove them
+        for(let i = items.length - 1; i >= 0; --i) {
+          const item = items[i];
+          if(item.isValid && field && !item.isValid(args.propertyRecord, field)) {
```

Comment:
althought its correct syntax, is confusing to use item.isValid in a condition this way as the fact that it is a function isn't obvious, it reads as though it could be a boolean and that makes the logic of this condition confusing to read. 

So doing `item.isValid !== undefined` is more clear